numpy/lib/type_check.py

```diff
@@ -327,6 +327,13 @@ def nan_to_num(x):
     ----------
     x : array_like
         Input data.
+    copy : bool, optional
+        .. versionadded:: 1.13
```

Comment:
I moved tag after the description. It looks like the tags in `numpy/lib/stride_tricks.py` may be wrong then (where I had followed the use there).